src/tape/ensemble.py

```diff
@@ -1237,6 +1240,165 @@ def _standardize_batch(self, batch, on, by_band):
+                frames_to_save.append("source")
+        else:
+            raise ValueError("Invalid input to `additional_frames`, must be boolean or list-like")
+
+        # Save the frame list to disk
+        for frame_label in frames_to_save:
+            # grab the dataframe from the frame label
+            frame = self.frames[frame_label]
+
+            # Object can have no columns, which parquet doesn't handle
+            # In this case, we'll avoid saving to parquet
+            if frame_label == "object":
+                if len(frame.columns) == 0:
+                    print("The Object Frame was not saved as no columns were present.")
+                    continue
+
+            # creates a subdirectory for the frame partition files
+            frame.to_parquet(os.path.join(ens_path, frame_label), **kwargs)
+
+        # Save a ColumnMapper file
+        col_map = self.make_column_map()
+        np.save(os.path.join(ens_path, "column_mapper.npy"), col_map.map)
+
+        print(f"Saved to {os.path.join(path, dirname)}")
+
+        return
+
+    def from_ensemble(self, dirpath, additional_frames=True, column_mapper=None, **kwargs):
+        """Load an ensemble from an on-disk ensemble.
+
+        Parameters
+        ----------
+        dirpath: 'str' or path-like, optional
+            A path to the top-level ensemble directory to load from.
+        additional_frames: bool, or list, optional
+            Controls whether EnsembleFrames beyond the Object and Source Frames
+            are loaded from disk. If True or False, this specifies whether all
+            or none of the additional frames are loaded. Alternatively, a list
+            of EnsembleFrame names may be provided to specify which frames
+            should be loaded. Object and Source will always be added and do not
+            need to be specified in the list. By default, all frames will be
+            loaded.
+        column_mapper: Tape.ColumnMapper object, or None, optional
+            Supplies a ColumnMapper to the Ensemble, if None (default) searches
+            for a column_mapper.npy file in the directory, which should be
+            created when the ensemble is saved.
+
+        Returns
+        ----------
+        ensemble: `tape.ensemble.Ensemble`
+            The ensemble object.
+        """
+
+        # First grab the column_mapper if not specified
+        if column_mapper is None:
+            map_dict = np.load(os.path.join(dirpath, "column_mapper.npy"), allow_pickle="TRUE").item()
+            column_mapper = ColumnMapper()
+            column_mapper.map = map_dict
+
+        # Load Object and Source
+        obj_path = os.path.join(dirpath, "object")
+        src_path = os.path.join(dirpath, "source")
+
+        # Check for whether or not object is present, it's not saved when no columns are present
+        if "object" in os.listdir(dirpath):
+            self.from_parquet(src_path, obj_path, column_mapper=column_mapper, **kwargs)
```

Comment:
Should we always use `sync_tables=False` since we synced object and source before saving?

Also as a general question that may be out of scope for this PR, are we preserving divisions here? If so I guess it would be helpful to use `sorted=True` if we knew that information?

Reply:
I believe we can always have `sync_tables=False`, right, good point!

For divisions, these are not being preserved at the moment, which definitely seems like an issue. We can save an ensemble without divisions being known, so the sorted parameter will need to be dependent on that information. 
Similar to above, maybe there's a need for a metadata file that should be generated to indicate whether divisions are known that can be read in?

Reply:
I just pushed another commit to better expose the sort/sorted kwargs from ensemble.from_parquet so they are usable in these cases. Some kind of metadata may be preferable here still so the user doesn't need to specify at all.

Reply:
Another update, I now have this working with metadata. ensemble.save_ensemble now stores a boolean per frame in the metadata.json file indicating whether that frame had divisions set. Instead of using the sort/sorted kwargs, I've changed ensemble.from_ensemble to use this to determine whether to calculate divisions or not. In the case of object and source, this is just by setting the sorted flag or not. For the other frames, the sorted flag is only applicable to the set_index call that ensemble.from_parquet folds into. So this PR now has the frames generate a parquet metadata file which populates divisions information if it's available, and the reading of additional frames now uses the parquet metadata file. 

I experimented with trying to get our ensemble.from_parquet to use these _metadata files, but it would be a non-trivial change in the logic that felt out of scope for this PR. It's potentially worth logging for the future as using the _metadata files would avoid having to calculate the min/max values per partition to get divisions, but on the other hand these files can get quite large for large datasets so they may have issues at scale.